var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -53,6 +53,7 @@ class SuperluDist(CMakePackage, CudaPackage):
 
     patch('xl-611.patch', when='@:6.1.1 %xl')
     patch('xl-611.patch', when='@:6.1.1 %xl_r')
+    patch('superlu-cray-ftn-case.patch', when='%cce')
```

Comment:
@xiaoyeli @pghysels - perhaps this fix can go into superlu-dist sources.

And in spack - limit this patch to version 7.1.1 and lower?

Reply:
Yes, it should go in SuperLU-dist sources.
I fixed the same issue in STRUMPACK last week (too late for 6.1.0) https://github.com/pghysels/STRUMPACK/commit/b6a0fb02db57453fa04cef24ef29d122e5b8c181
(I initially tried to install *.mod using a wildcard, but that also installs empty folders, which led to infinite recursion when doing an in-source build: https://github.com/pghysels/STRUMPACK/commit/390835f92378466cc9c248680f4195056777eb8f

Reply:
for now - perhaps:

```suggestion
    patch('superlu-cray-ftn-case.patch', when='@:7.1.1 %cce')
```